install/installer/pkg/components/workspace/ide/constants.go

```diff
@@ -6,7 +6,7 @@ package ide
 
 const (
 	CodeIDEImage                = "ide/code"
-	CodeIDEImageStableVersion   = "commit-80d9b1ebfd826fd0db25320ba94d762b51887ada" // stable version that will be updated manually on demand
+	CodeIDEImageStableVersion   = "commit-15cce5f092604dd30df24e86458222815bc73aca" // stable version that will be updated manually on demand
```

Comment:
I think this tag is wrong, the correct one is this `commit-9c6caab7bf86403da9250db7a13b9b7873cd6d98`  from https://werft.gitpod-dev.com/job/gitpod-build-afalz-generate-code-stable-1.68.1.

Reply:
`15cce5f092604dd30df24e86458222815bc73aca` is the commit hash of the commit in `main` generated by merging https://github.com/gitpod-io/gitpod/pull/10567.

`9c6caab7bf86403da9250db7a13b9b7873cd6d98` is the commit of the PR which then was squashed.

Could you clarify why choosing one over the other?

Reply:
Ah it's fine then :smile: it's just I was already used to use the one from the PR

Reply:
I think we can choose a faster one. And maybe we need to make a deal about which to choose to avoid confusion for later changer.

- hash from main - its build sometimes will be failed, as image below
- hash from prev PR - build must be success before merge and we don't wait for another round of **painful** waiting

That's why I prefer prev PR one.

|Image|
|-|
|<img width="501" alt="image" src="https://user-images.githubusercontent.com/20944364/173000009-93f0c796-6436-462a-9fd1-7e6e34ec7c04.png">|

Reply:
agreed 👍